### PR TITLE
Fix #2333: Prevent crash on Administrator Controls when it is selected again in navigation drawer

### LINKED ISSUE #2333: Click on Admin Controls crash in Admin Controls
**Describe the bug**
If we are in `Administrator Controls` screen and select `Administrator Controls` from Navigation Drawer again, it crashes the app.

**To Reproduce**
1. Select `Admin` profile.
2. Open `Navigation Drawer`
3. Select `Administrator Controls`
4. Open `Navigation Drawer`
5. Select `Administrator Controls`
The application will crash.

**Expected behavior**
The application should not crash and the navigation drawer should be closed.

**Screenshot**
Screenshot 2020-12-29 at 11 26 48 AM

### PR DESCRIPTION
**Pull request by ShridharGoel**
## Explanation
Fixes #2333 

## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

## Screenshots
Screenshot 1
Screenshot 2

### CODE REVIEW

**Comment by anandwana001** [comment]
Need to add a test case covering this flow.

**Comment by ShridharGoel** [comment]
@anandwana001 Added test case along with screenshot.

File: app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt [hunk 1]
```diff
@@ -100,6 +100,11 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
         getHeaderViewModel().profile.set(it)
         getFooterViewModel().isAdmin.set(it.isAdmin)
         binding.administratorControlsLinearLayout.setOnClickListener {
+          if (getFooterViewModel().isAdministratorControlsSelected.get() == true) {
```
Comment (anandwana001) [inline app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt:103]: isn't `if (getFooterViewModel().isAdministratorControlsSelected.get()!!) {` this is good to go?

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt (hunk 1 above)
Comment (ShridharGoel) [reply app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt:103]: No, in that case !! operator will be required.

```
if (getFooterViewModel().isAdministratorControlsSelected.get()!!) {

}
```

**Comment by anandwana001** [comment]
Also, remove the function `createNavigationDrawerActivityIntent` from test file.

**Comment by ShridharGoel** [comment]
@anandwana001 Done.

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @ShridharGoel

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**